components/PlantPage.js

```diff
@@ -0,0 +1,15 @@
+
+import PlantList from './PlantList';
+
+
+
+function PlantPage({plants}) {
+  return (
+    <div>
+      <PlantList plants={plants} />
+    </div>
+  );
+}
+
+export default PlantPage;
+
```

Comment:
Since this file only renders the `PlantList`-component, there is no need for it. You can render the `PlantList`-component directly from the index.js.